index.html

```diff
@@ -543,7 +543,7 @@ <h1>
 
 <p>To link directly to the latest release, copy this snippet:
 
-<pre><code class="html">&lt;script src="https://d3js.org/d3.v5.min.js">&lt;/script></code></pre>
+<pre><code class="html">&lt;script src="https://d3js.org/d3.v6.min.js">&lt;/script></code></pre>
```

Comment:
Might it be wise to use a well known CDN here instead? Maybe 

https://cdn.jsdelivr.net/npm/d3@6.0.0/dist/d3.min.js

or

https://unpkg.com/d3@6.0.0/dist/d3.min.js

?

I often wonder about the tradeoffs around how to load the D3 script. Maybe https://d3js.org/d3.v6.min.js is indeed the best option, but not sure really about pros and cons.